jsonschema-validation.xml

```diff
@@ -896,6 +896,15 @@
                         <xref target="RFC6901">RFC 6901, section 5</xref>
                     </t>
                 </section>
+                <section title="regex">
+                    <t>
+                        This attribute applies to string instances.
+                    </t>
+                    <t>
+                        A regular expression, which SHOULD be valid according to the
+                        <xref target="ecma262">ECMA 262</xref> regular expression dialect.
```

Comment:
I don't recall off hand, but I suspect strongly that the original spec made the same mention here that implementations may not be using ECMA regexes (and that this format might actually mean "*some* regex dialect, so restrict yourself to the common subsets).

Reply:
Thanks @Julian, I'd forgotten about that section.  I've updated it and made the regex definition reference it.